manifests/portworx-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: portworx
```

Comment:
Although Portworx is disabled by default, this namespace will still be created. Perhaps, it should be commented for now?

Reply:
Commented out for now.